includes/gateway/class-omise-payment-alipay.php

```diff
@@ -61,95 +61,47 @@ public function init_form_fields() {
 		}
 
 		/**
-		 * @param  int $order_id
-		 *
-		 * @see    WC_Payment_Gateway::process_payment( $order_id )
-		 * @see    woocommerce/includes/abstracts/abstract-wc-payment-gateway.php
-		 *
-		 * @return array
+		 * @inheritdoc
 		 */
-		public function process_payment( $order_id ) {
-			if ( ! $order = $this->load_order( $order_id ) ) {
-				wc_add_notice(
-					sprintf(
-						wp_kses(
-							__( 'We have been unable to process your payment.<br/>Please note that you\'ve done nothing wrong - this is likely an issue with our store.<br/><br/>Feel free to try submitting your order again, or report this problem to our support team (Your temporary order id is \'%s\')', 'omise' ),
-							array(
-								'br' => array()
-							)
-						),
-						$order_id
-					),
-					'error'
-				);
-				return;
-			}
+		public function charge( $order_id, $order ) {
+			$metadata = array_merge(
+				apply_filters( 'omise_charge_params_metadata', array(), $order ),
+				array( 'order_id' => $order_id ) // override order_id as a reference for webhook handlers.
+			);
 
-			$order->add_order_note( __( 'Omise: Processing a payment with Alipay solution..', 'omise' ) );
+			return OmiseCharge::create( array(
```

Comment:
Any reason why using long array syntax? WPress guidelines say either is fine and suggest moving to short if no requirement for backwards compatibility

Reply:
https://make.wordpress.org/core/2019/03/26/coding-standards-updates-for-php-5-6/

Reply:
@jonrandy  To make its code style conform with the rest of our code base.
Omise-WooCommerce has been using long array syntax for entire project so I am avoiding making any confusion (we may replace all long array syntax to a short one later on if necessary)